src/test/java/com/pi4j/crowpi/components/SoundSensorComponentTest.java

```diff
@@ -0,0 +1,61 @@
+package com.pi4j.crowpi.components;
+
+import com.pi4j.crowpi.ComponentTest;
+import com.pi4j.io.gpio.digital.DigitalInput;
+import com.pi4j.io.gpio.digital.DigitalState;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SoundSensorComponentTest extends ComponentTest {
+    protected SoundSensorComponent soundSensor;
+    protected DigitalInput din;
+
+    @BeforeEach
+    public void setUp() {
+        soundSensor = new SoundSensorComponent(pi4j);
+        din = soundSensor.getDigitalInput();
+    }
+
+    @Test
+    public void testGetSensorState() {
+        // given
+        SoundSensorComponent.SoundState resultState;
+
+        // when
+        resultState = soundSensor.getState();
```

Comment:
```suggestion
        // when
        final var resultState = soundSensor.getState();
```

I think we can initialize the variable within the `// when` section and only use `// given` if we actually setup any state or environment before the test.